common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,147 @@
+	</div>
+	<div>
+		<h2><code>.small</code></h2>
+		<dd>Decreases the font size of the element.</dd>
+		<p class="small">
+			Example small text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;small&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.mark</code></h2>
+		<dd>Highlights the text</dd>
+		<p class="mark">
+			Example highlighted text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;mark&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.lead</code></h2>
+		<dd>Increases the font size and line height</dd>
+		<p class="lead">
+			Example lead text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;lead&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.text-*</code></h2>
+		<dd>Can be used to align text or change its case.</dd>
+		<h2>Alignment examples:</h2>
+		<div class="row">
+			<p class="text-left col-sm-6">
+				Aligned left
+			</p>
+			<pre class="text-left col-sm-6"><code>&lt;p <strong>class=&quot;text-left&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-right col-sm-6">
+				Aligned right
+			</p>
+			<pre class="text-right col-sm-6"><code>&lt;p <strong>class=&quot;text-right&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-center col-sm-6">
+				Aligned center
+			</p>
+			<pre class="text-center col-sm-6"><code>&lt;p <strong>class=&quot;text-center&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-justify col-sm-6">
+				Justified text
+			</p>
+			<pre class="text-justify col-sm-6"><code>&lt;p <strong>class=&quot;text-justify&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-nowrap col-sm-6">
+				No-wrap text: xxxxxxxxxxxxxxxxxxxxxxx
+			</p>
+			<pre class="text-nowrap col-sm-6"><code>&lt;p <strong>class=&quot;text-danger&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+
+		<h2>Transformation examples:</h2>
+		<div class="row">
+			<p class="text-lowercase col-sm-6">
+				LOWERCASE TEXT
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-lowercase&quot;</strong>&gt;LOWERCASE TEXT&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-uppercase col-sm-6">
+				uppercase text
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-uppercase&quot;</strong>&gt;uppercase text&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-capitalize col-sm-6">
+				capitalized text
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-capitalize&quot;</strong>&gt;capitalized text&lt;/p&gt;</code></pre>
+		</div>
+	</div>
+	<div>
+		<h2><code>.no-wrap</code></h2>
+		<dd>Prevents the text from wrapping</dd>
+		<p class="no-wrap">
```

Comment:
There is a typo, there is no dash

```suggestion
		<p class="nowrap">
```

See: https://github.com/wet-boew/wet-boew/blob/442cc90ece58e548682b402d1553a7cfac1e2bd3/src/base/views/_screen.scss#LL66-L72C2